packages/js/src/utils/swc/compile-swc.ts

```diff
@@ -0,0 +1,64 @@
+import { TypeScriptCompilationOptions } from '@nrwl/workspace/src/utilities/typescript/compilation';
+import { exec, execSync } from 'child_process';
+import { normalizeTsCompilationOptions } from '../normalize-ts-compilation-options';
+
+export async function compileSwc(
+  tsCompilationOptions: TypeScriptCompilationOptions,
+  postCompilationCallback: () => void | Promise<void>
+): Promise<{ success: boolean }> {
+  const normalizedOptions = normalizeTsCompilationOptions(tsCompilationOptions);
+
+  console.log(`Compiling with SWC for ${normalizedOptions.projectName}...`);
+  const srcPath = normalizedOptions.projectRoot;
+  const destPath = normalizedOptions.outputPath.replace(
+    `/${normalizedOptions.projectName}`,
+    ''
+  );
+  const swcrcPath = `${normalizedOptions.projectRoot}/.swcrc`;
+
+  // TODO(chau): use `--ignore` for swc cli to exclude spec files
+  // Open issue: https://github.com/swc-project/cli/issues/20
+  let swcCmd = `npx swc ${srcPath} -d ${destPath} --source-maps --config-file=${swcrcPath}`;
+
+  if (normalizedOptions.watch) {
+    swcCmd += ' --watch';
+    return createSwcWatchProcess(swcCmd, postCompilationCallback);
+  }
+
+  const swcCmdLog = execSync(swcCmd).toString();
+  console.log(swcCmdLog.replace(/\n/, ''));
```

Comment:
Remove this?

Reply:
This one is to log what `swcCmd` logs which is to show how many files were compiled by SWC.